src/components/pager/pager.test.js

```diff
@@ -589,4 +484,266 @@ context("Test for Pager component", () => {
       currentPageWrapper().should("have.css", "margin-bottom", "0px");
     });
   });
+
+  describe("Accessibility tests for Pager component", () => {
+    it.each([[2], [5], [7]])(
+      "should render Pager with currentPage prop set to %s for accessibility tests",
+      (currentPage) => {
+        CypressMountWithProviders(<PagerComponent currentPage={currentPage} />);
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each([[50], [100], [235]])(
+      "should render Pager with totalRecords prop set to %s for accessibility tests",
+      (totalRecords) => {
+        CypressMountWithProviders(
+          <PagerComponent totalRecords={totalRecords} />
+        );
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each([1, 10, 25, 100])(
+      "should render Pager with pageSize prop set to %s for accessibility tests",
+      (pageSizeVal) => {
+        CypressMountWithProviders(
+          <PagerComponent pageSize={pageSizeVal} showPageSizeSelection />
+        );
+        cy.checkAccessibility();
+      }
+    );
+    it("should render Pager with pageSizeSelectionOptions prop for accessibility tests", () => {
+      CypressMountWithProviders(
+        <PagerComponent
+          pageSizeSelectionOptions={recordsDiff}
+          showPageSizeSelection
+        />
+      );
+      cy.checkAccessibility();
+    });
+
+    it.each([true, false])(
+      "should render Pager with showPageSizeSelection prop set to %s for accessibility tests",
+      (bool) => {
+        CypressMountWithProviders(
+          <PagerComponent showPageSizeSelection={bool} />
+        );
+        cy.checkAccessibility();
+      }
+    );
+
+    // FE-4659
+    describe.skip("should render Pager component for accessibility tests", () => {
```

Comment:
why we are skipping all those tests? I can't see the accessibility violations on Storybook - but have the same config file